modules/installation-configuration-parameters.adoc

```diff
@@ -384,6 +384,16 @@ ifdef::gcp[]
 |The name of the existing VPC that you want to deploy your cluster to.
 |String.
 
+|`platform.gcp.type`
+|The link:https://cloud.google.com/compute/docs/machine-types[GCP machine type].
+|The subnet name.
```

Comment:
> |The subnet name.
Does it mean the description for platform.gcp.type? If so, please use The GCP machine type instead